src/components/FormLayout/__tests__/FormLayout.spec.tsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import {
+  expect,
+  test,
+} from '@playwright/experimental-ct-react';
+import { propTests } from '../../../../tests/playwright';
+import {
+  FormLayoutForTest,
+  FormLayoutWithCustomFieldForTest,
+  FormLayoutWithoutChildrenForTest,
+} from './FormLayout.story';
+import { fieldWidthPropTest } from './_propTests/fieldWidthPropTest';
+import { fieldLayoutPropTest } from './_propTests/fieldLayoutPropTest';
+import { autoWidthPropTest } from './_propTests/autoWidthPropTest';
+import { customFieldLabelAlignmentPropTest } from './_propTests/customFieldTests/customFieldLabelAlignmentPropTest';
+import { customFieldValidationStatesPropTest } from './_propTests/customFieldTests/customFieldValidationStatesPropTest';
+import { customFieldInnerFieldSizePropTest } from './_propTests/customFieldTests/customFieldInnerFieldSizePropTest';
+import { customFieldRequiredPropTest } from './_propTests/customFieldTests/customFieldRequiredPropTest';
+
+test.describe('FormLayout', () => {
+  test.describe('base', () => {
+    test.describe('visual', () => {
+      [
+        ...propTests.defaultComponentPropTest,
+        ...fieldWidthPropTest,
```

Comment:
See the snapshots. All the output look same. Please. try to find such setup that will levarage the usage and test FormLayout limitations if it have some. Also, think about mixin layout with other props to verify if it works correctly. See the documentation for allowed variants just to be sure what we need to test.